File: app/src/main/java/com/abs/launcher/AppInterface.java
package com.abs.launcher;

import android.app.Application;

/**
 * Created by ZY on 2017/4/2.
 */

public class AppInterface {

    static AppInterface sInstance;
    static Application sApplication;

    LauncherModel model;

    static AppInterface getApp() {
        return sInstance;
    }

    public static Application getApplication() {
        return sApplication;
    }

    public LauncherModel getModel() {
        return model;
    }
}

File: app/src/main/java/com/abs/launcher/AppInterface.kt
package com.abs.launcher

import android.app.Application

/**
 * Created by zy on 18-1-3.
 */
class AppInterface {

    var model: LauncherModel? = null

    var application: Application? = null

    companion object {
        var instance: AppInterface? = null
    }
}